packages/react/context-menu/src/ContextMenu.tsx

```diff
@@ -84,32 +87,26 @@ type ContextMenuContentOwnProps = {
 
 const ContextMenuContent = forwardRefWithAs<typeof MenuPrimitive.Root, ContextMenuContentOwnProps>(
   (props, forwardedRef) => {
-    const {
-      anchorRef,
-      disableOutsidePointerEvents = true,
-      side = 'bottom',
-      align = 'start',
-      ...contentProps
-    } = props;
+    const { anchorRef, ...contentProps } = props;
```

Comment:
Changes in this file made me realise this `anchorRef` prop isn't being used. Is this a bug?